src/shadowbox/server/manager_metrics.ts

```diff
@@ -15,8 +15,21 @@
 import {PrometheusClient} from '../infrastructure/prometheus_scraper';
 import {DataUsageByUser, DataUsageTimeframe} from '../model/metrics';
 
+export type TunnelTimeDimension = 'access_key' | 'country' | 'asn';
+
+interface TunnelTimeRequest {
+  hours: number;
+}
+
+interface TunnelTimeResponse {
+  location?: string;
+  asn?: number;
+  tunnel_time: number;
```

Comment:
Use a suffix to indicate the unit of duration. Loosely following https://google.aip.dev/142 I'm inclined to suggest `tunnel_time_seconds`, using seconds instead of hours as the unit.